authentication/LegacyAuthQuickstart/AuthenticationExample/AppDelegate.m

```diff
@@ -35,11 +35,6 @@ - (BOOL)application:(UIApplication *)application
   [FIRApp configure];
   // [END firebase_configure]
 
-  // [START setup_gidsignin]
```

Comment:
@morganchen12 I'm not sure if the tags are affecting anything and if I updated them properly. Could you PTAL.

Reply:
Deleting this snippet is ok.

Reply:
SG. Please add me to reviewers for the updated docs to ensure they have correct snippets.